students/GuyCChan/session09/test_objOriMailRoom.py

```diff
@@ -0,0 +1,226 @@
+from objOriMailRoom import Donor, DonorCollection, make_report, letters_to_donors, test_data
+import pytest
+
+# Tests for Donor class.
+def test_init():
+    db = donors_db()
+    assert len(donors_db) == 0
+
+def test_Donor_in_db():
+    donor = donor("Mike May")
+    assert donor.name == "Mike May"
+    assert donor.donation is None
```

Comment:
a Donor may have had multiple donations -- so what does donor.donation mean? the last one? all of them, ? 

having it be None at this stage is fine, but what it the end goal for this attribute (or property)?